__tests__/__fixtures__/tx_history.js

```diff
@@ -14,7 +14,7 @@ export default [
         decoded: {
           type: "P2PKH",
           address: "WYiD1E8n5oB9weZ8NMyM3KoCjKf1KCjWAZ",
-          timelock: 1713662137,
```

Comment:
This was set to 2024, it would eventually break the CI so I pushed it to the year 3000.